diagnostics/src/main.rs

```diff
@@ -24,24 +25,30 @@ enum Commands {
 
 #[derive(Debug, Args)]
 struct CommonOpts {
+    /// Kafka message broker, should have format `host:port`, e.g. `localhost:19092`
     #[clap(long)]
     broker: String,
 
+    /// Optional Kafka username
     #[clap(long)]
     username: Option<String>,
 
+    /// Optional Kafka password
     #[clap(long)]
     password: Option<String>,
 
+    /// Kafka consumer group e.g. --kafka_consumer_group trace-producer
     #[clap(long = "group")]
     consumer_group: String,
 
+    /// The Kafka topic that trace messages are produced to
     #[clap(long)]
     topic: String,
 }
 
 #[derive(Debug, Args)]
 struct DaqTraceOpts {
+    /// The interval at which the message rate is calculated. A lower value will result in the message rate being calculated more frequently.
```

Comment:
```suggestion
    /// The interval at which the message rate is calculated.
```

What unit?

Reply:
Seconds. I will add this in, but maybe this was a bad choice on my part since most of the command line arguments use milliseconds to represent time.